packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div id="new-collaborators-form" data-testid="new-collaborators-form">
+  <div
+    id="new-collaborators-form"
+    data-testid="new-collaborators-form"
+    data-custom-key-bindings="true"
```

Comment:
Can't we just set the `data-custom-key-bindings="true"` on the `SideBar.vue` component?